eng/testing/tests.mobile.targets

```diff
@@ -101,11 +117,36 @@
         <Output TaskParameter="AppBundlePath" PropertyName="AppBundlePath" />
         <Output TaskParameter="XcodeProjectPath" PropertyName="XcodeProjectPath" />
     </AppleAppBuilderTask>
+
+    <ItemGroup>
+      <_appFiles Include="$(AppBundlePath)/../**/*" />
```

Comment:
If you had a path (or paths) to an AppBundle you want to run on iOS through Helix, you could skip the whole part of creating own Helix call (and payload scripts/zipping..) and set the `XHarnessiOSProject` task item list from Arcade and everything will be handled for you.

There is an example in the Arcade repository:
https://github.com/dotnet/arcade/blob/master/tests/UnitTests.XHarness.iOS.proj

The point of this is that it is a bit tricky to run the XHarness tool because of some MacOS caveats and we already resolved that (and will keep it working) in Arcade. This way, there's a duplication of the payload shell script and the whole process of sending the zipped folder and getting the results back.

So it would be better to use the Helix SDK from Arcade and let it do all the work.

But maybe there are some reasons for you not doing that and I don't know about those so let's sync and talk about it.

Reply:
@safern What was the rationale for dotnet/runtime not using the helix SDK?

Reply:
Unfortunately we can't use the helix built-in support to produce the app bundle since we use our custom runtime-pack since we build our own shared framework. Could you elaborate what the process of producing the app bundle is for the Helix SDK?

We produce the app bundles as part of our test build (since we also need to support this locally, for developers to run tests in their dev machines).

Reply:
@safern this is not about producing the app bundle but about how we construct the Helix job payload and run it through the Helix which the SDK provides. The Helix SDK cannot create the app bundle (yet?) but it can run the app bundle you give it in Helix and process the results which is the part that should really be done through the SDK.

However, the way the packaging in XHarness was designed (and it supports Xamarin.iOS but not .NET 5 yet), it is able to bundle the correct runtime inside as well, I don't see a problem with that. But not the question at hand..